wordpress-mobile/WordPress-iOS
# Posts & Pages: New design for pages cells

Fixes #21820, removes the Undo button, and update the Templates design.

Known issues:

- The "Published/Edited" part is missing from the date label (should be done together with the Posts)
- The list doesn't update when you create a new page (hope it's not a regression; I put it on the list)
- The custom homepage is not displayed https://github.com/wordpress-mobile/WordPress-iOS/issues/21841

To test:

- Create a new site with a FSE theme and verify that the Homepage template cell is displayed and has an updated design
- Verify that "regular" pages have an updated design

## Regression Notes
1. Potential unintended areas of impact: Pages List
2. What I did to test those areas of impact (or what existing automated tests I relied on): manual
3. What automated tests I added (or what prevented me from doing so): n/a

PR submission checklist:


UI Changes testing checklist:
